frontend/components/App/App.tsx

```diff
@@ -77,6 +81,17 @@ const App = ({ children, location }: IAppProps): JSX.Element => {
         const noSandboxHosts = hostCount.count === 0;
         setNoSandboxHosts(noSandboxHosts);
       }
+
+      // TODO: Confirm with MDM what to check on config to not hit 404 for /abm API
```

Comment:
@ghernandez345 @gillespi314 Just wanted to confirm what to check on the config to not call the /abm and /apns APIs and hit a 404. My best guess was only call the APIs if `configResponse.mdm.apple_bm_enabled_and_configured`

Reply:
can we just catch the error and assume that means it's not configured?

Reply:
or parse it for certain message, which if present, means it's not configured otherwise render an error?